README.md

```diff
@@ -26,6 +26,20 @@ main = launchAff_ do
   log response.body
 ```
 
+## Effect and Aff
+
+`Effect` is a synchronous effect monad. It is used to sequence effectful foreign (JavaScript) code — things like random number generation, reading and writing mutable values, writing to the console and throwing and catching exceptions.[<sup>1</sup>](https://stackoverflow.com/questions/37661391/what-are-eff-and-aff)
+
+`Aff` is an asynchronous effect monad. It can handle and sequence effectful asynchronous code, like AJAX requests, timeouts, and network and file IO. 
```

Comment:
```suggestion
`Aff` is an asynchronous effect monad. It can handle and sequence effectful asynchronous code, like HTTP requests, timeouts, and network and file IO. 
```

I know we have Affjax, but AJAX is just not a term that is used anymore. I think we should keep it in the early 2000s 😄.

Reply:
lol ok

Reply:
Done.